src/en/editorial-content/general.md

```diff
@@ -2,48 +2,45 @@
 title: "General recommendations for content"
 ---
 
-# General recommendations for content
+# General recommendations 
 
 ## Introduction
-This section proposes general recommendations to ensure your contents are accessible whatever the medium used (email, Web, Word, etc.).
+This section presents general recommendations to follow to ensure the accessibility of your content regardless of the medium used (email, PPT, PDF, Word, etc.).
 
 ## Check colours
 
-- Make sure colour is not the only means used to convey information.
-- Make sure there is sufficient contrast between the text colour and the background.
-
-The contrast can be checked with <a href="https://www.paciellogroup.com/resources/contrastanalyser/" hreflang="en" lang="en">Colour Contrast Analyzer</a>&nbsp;:
- - 4.5:1 for normal size text.
- - 3:1 for big text.
-
-## Facilitate reading
-
-- Align text to the left, don’t justify it.
-- Use an easily-readable font family with a minimum size of 12 pixels.
-- Use simple punctuation.
-- Avoid italics, completely capitalised sentences; but always put a capital at the beginning of a sentence.
-- Avoid textured backgrounds (images).
-- Do not do too many repeated carriage returns or tabs to space contents (rather, use “Paragraph&nbsp;&gt; Spacing” or “Indent” in Word for instance).
-- Avoid layout and data tables.
-- Avoid information-conveying content in images (graphs, schemas, diagrams,…)
-- Signal the end of the document (for instance, an image saying “End of email”, or a white-on-white text “End of document”).
-- Use simple, clear and short phrase turns: subject, verb, complement. One idea per sentence, with simple words.
-- Explain complicated words.
-- Use the present tense, preferably direct style, and active form (no subjunctive, no conjunctions).
-- Use proper typographic characters: dash -, em-dash —, “rounded quotes”.
-<!-- removed 1 line re: accents and caps – not relevant in English --> 
-- Avoid foreign words, abbreviations except those that are genuinely used.
-
-## Provide accessible layout tables
-
-Simplify tables, even if we know that email clients (including Outlook) still often demand to use tables for layout.
-
-Checkpoints:
-
-- In general, don’t do complex table layouts (merging cells, multicolumn,…).
-- Avoid embedding layout tables (maximum two levels of embedding), but also:
-- Don’t merge table cells, but use several simpler tables.
-- Don’t keep empty cells.
-- Don’t add background images.
-- Make sure that the sequential order is correct (one can read left to right and top to bottom).
-- Avoid complex data tables, graphs or schemas.
+* Make sure [color is not the only means used to communicate information](/en/web/design/colors-and-contrasts/#do-not-use-colour-or-sensory-characteristics-as-the-unique-source-of-information).
```

Comment:
Set phrase: “to convey information”
s/communicate/convey/

Reply:
Fixed in c216677282df01aa3e077a45ded6d6e8e62eff4d